protocols/gossipsub/src/metrics.rs

```diff
@@ -133,10 +133,6 @@ pub(crate) struct Metrics {
     ignored_messages: Family<TopicHash, Counter>,
     /// The number of messages rejected by the application (validation result).
     rejected_messages: Family<TopicHash, Counter>,
-    /// The number of publish messages dropped by the sender.
-    publish_messages_dropped: Family<TopicHash, Counter>,
-    /// The number of forward messages dropped by the sender.
-    forward_messages_dropped: Family<TopicHash, Counter>,
```

Comment:
These were kind of useful, but if we can't have them in the new regime, all g

Reply:
we can still have them, wdyt of recording all of the types of messages lost instead?